### PR TITLE
Create desktop version of Portfolio Website

### PR DESCRIPTION
**Pull request by Progress2002**
### Modify index.html file

- Add links to the `nav` section
- Add desktop image to the work section

### Modify styles.css file

- Add styles to style the desktop layout of the webpage
- Applied downloaded Figma desktop images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,135 +78,138 @@ <h2>I’m a software developer</h3>
...
+                            </p>
+                            <ul>
+                                <li>css</li>
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <div class="btn">
+                        <button>See Project</button>
```
Comment (ichala) [inline index.html]: - [x] Please note that the `<button>See Project</button>` should be a link that can take for example to the project repo in GitHub , kindly wrap your button inside an anchor tag 🙏

File: index.html [hunk 2]
```diff
@@ -58,135 +78,138 @@ <h2>I’m a software developer</h3>
...
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button>See Project</button>
+                    </div>
+                </li>
+            </ul>
+        </div>
```
Comment (ichala) [inline index.html]: - [x] Please note that the first project card should be the same as all the others in the default/intial state , only when the user hovers on any card , does the description and dark opacity should disappears and a See Project button appears at the bottom of the image , kindly fix that 🙏 


image

File: index.html [hunk 3]
```diff
@@ -58,135 +78,138 @@ <h2>I’m a software developer</h3>
...
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button>See Project</button>
+                    </div>
```
Comment (ichala) [inline index.html]: - [x] same thing here kindly make sure all the `<button>See Project</button>` buttons are wrapped in an anchor tag , kindly fix them 🙏

File: index.html [hunk 4]
```diff
@@ -58,135 +78,138 @@ <h2>I’m a software developer</h3>
...
-            </div>
-            <div>
-                <hr>
-            </div>
-        </div>
-        <ul id="work-content">
-            <li class="project pr-1">
-                <div class="image">
-                    <img src="images/Img Placeholder.png" alt="img">
+    <section id="work-section">
```
Comment (ichala) [inline index.html:81]: - [ ] please note that the work section should have a white background instead of `#e5e5e5` kindly fix it to match figma design

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Progress2002 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great design ✔️ 
- Clean Code ✔️

 ## Required Changes ♻️

- [ ] Check the comments below

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Progress2002) [reply index.html]: Thanks for the observation 🙏🏼 corrections have been made

File: index.html (hunk 4 above)
Comment (Progress2002) [reply index.html:81]: Thanks!  Changes have been made

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED 
 ## Hi @Progress2002,

 Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: **Congratulations! tada**
 
  
## Highlights:
 * Well written PR description :heavy_check_mark:
 * Proper use of GitHub flow :heavy_check_mark:
 * All tests passed :heavy_check_mark:

 
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._
 
 Cheers and Happy coding! :clap: :clap: :clap:
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**